servicesContracts.py

```diff
@@ -1,10 +1,28 @@
 from typing import Any
-
-from artifacts import *
 from servicesContracts.requestEthereum_service import RequestEthereumService
 
+"""
+  getServiceFromAddress return the service of a coresponding currency contract address
+  :param  address:
+        The address of the currency contract
+"""
 def getServiceFromAddress(address: str) -> Any:
-    pass
+    if not address:
+        return None
+    if isThisArtifact(Artifacts.requestEthereumArtifact, address):
```

Comment:
You should be able to just use `requestEthereumArtifact` instead of `Artifacts.requestEthereumArtifact`.